docs/localdatetime.md

```diff
@@ -0,0 +1,1242 @@
+This range covers approximately half a million years. If `epochNanoseconds` is outside of this range, a `RangeError` will be thrown.
+
+Usage examples:
+
+```javascript
+// UNIX epoch in California
+new Temporal.LocalDateTime(0n, Temporal.TimeZone.from('America/Los_Angeles), Temporal.Calendar.from('iso8601'))
+  // => 1969-12-31T16:00-08:00[America/Los_Angeles]
+```
+
+## Static methods
+
+### Temporal.LocalDateTime.**from**(_thing_: any, _options_?: object) : Temporal.LocalDateTime
+
+**Parameters:**
+
+- `thing`: The value representing the desired date, time, time zone, and calendar.
+- `options` (optional object): An object with properties representing options for constructing this instance.
+  The following options are recognized:
+  - `overflow` (string): How to deal with out-of-range values in `thing`.
+    Allowed values are `constrain` and `reject`.
+    The default is `constrain`.
+  - `disambiguation` (string): How to disambiguate if the date and time given by `localDateTime` does not exist in the time zone, or exists more than once.
+    Allowed values are `'compatible'`, `'earlier'`, `'later'`, and `'reject'`.
+    The default is `'compatible'`.
+  - `offset` (string): How to interpret a provided time zone offset (e.g. `-02:00`) if it conflicts with the provided time zone e.g. `America/Sao_Paulo`.
+    Allowed values are `'use'`, `'ignore'`, `'prefer'`, and `'reject'`.
+    The default is `'reject'`.
+
+**Returns:** a new `Temporal.LocalDateTime` object.
+
+This static method creates a new `Temporal.LocalDateTime` object from another value.
+If the value is another `Temporal.LocalDateTime` object, a new object representing the same date and time is returned.
+If the value is any other object, a `Temporal.LocalDateTime` will be constructed from the values of any `timeZone`, `timeZoneOffsetNanoseconds`, `era`, `year`, `month`, `day`, `hour`, `minute`, `second`, `millisecond`, `microsecond`, `nanosecond`, and/or `calendar` properties that are present.
+At least the `timeZone`, `year`, `month`, and `day` properties must be present. Other properties are optional.
+If `calendar` is missing, it will be assumed to be `Temporal.Calendar.from('iso8601')`.
+Any other missing properties will be assumed to be 0 (for time fields).
+
+All date/time values will be interpreted in context of the provided time zone.
+Also, if `calendar` is present then all date/time values will also be interpreted in context of that calendar.
+
+Any non-object value is converted to a string, which is expected to be in an extension of ISO 8601 that includes a required time zone and optional calendar. For example:
+
+```
+2020-08-05T20:06:13+0900[Asia/Tokyo][c=japanese]
+```
+
+If the string isn't valid, then a `RangeError` will be thrown regardless of the value of `overflow`.
+
+Note that although this string format is not currently standardized by ISO 8601, it matches the format used by `java.time` and some other time-zone-aware libraries.
```

Comment:
Given the volume of feedback we've already received about developers concerned that we are diverging from the standard, I think it might be worth a sentence in the main docs, because many developers might not make it over to the extensions page.   But I agree we could make this smoother. If we did want to have some text here, along the lines of "We know it's not standardized yet but JS uses it and Java uses it and we're even working on making it a published standard, so don't be freaked out about this", what do you think would be a good way to finesse it?

Reply:
Maybe link explicitly to the ISO extensions page? Something like "For more information on Temporal's extensions to the ISO string format and the process of becoming a published standard, see 'ISO standard extensions'"

Reply:
That's a great solution. I'll use your text slightly modified: 

> For more information on `Temporal`'s extensions to the ISO string format and the progress towards becoming a published standard, see [ISO standard extensions](./iso-string-ext.md).